linux jdk doesn't support -d32

"All other platforms (Windows and Linux) contain separate 32 and 64-bit
installation packages. If both packages are installed on a system, you select
one or the other by adding the appropriate "bin" directory to your path. For
consistency, the Java implementations on Linux accept the -d64 option."

https://www.oracle.com/java/technologies/hotspotfaq.html#64bit_layering

--- a/unix/vncviewer/vncviewer.in
+++ b/unix/vncviewer/vncviewer.in
@@ -53,8 +53,4 @@
 	LD_LIBRARY_PATH=$JAWT_PATH:$LD_LIBRARY_PATH
 	export LD_LIBRARY_PATH
 fi
-if [ "@BITS@" = "32" ]; then
-	exec $JAVA -server -d@BITS@ -Djava.library.path=$TVNC_JAVADIR -jar $TVNC_JAVADIR/VncViewer.jar ${1+"$@"}
-else
-	exec $JAVA -server -Djava.library.path=$TVNC_JAVADIR -jar $TVNC_JAVADIR/VncViewer.jar ${1+"$@"}
-fi
+exec $JAVA -server -Djava.library.path=$TVNC_JAVADIR -jar $TVNC_JAVADIR/VncViewer.jar ${1+"$@"}
